packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -243,60 +243,49 @@ export default class Jobs {
   }
 
   async _runJob(job: Job, schedule?: Schedule, data_?: any) {
-    const { id } = job
-
-    const runningJobs = this._runningJobs
-    if (id in runningJobs) {
-      throw new Error(`job ${id} is already running`)
-    }
-
-    const { type } = job
-    const executor = this._executors[type]
-    if (executor === undefined) {
-      throw new Error(`cannot run job ${id}: no executor for type ${type}`)
-    }
-
-    let data
-    if (type === 'backup') {
-      // $FlowFixMe only defined for BackupJob
-      const settings = job.settings['']
-      data = {
-        // $FlowFixMe only defined for BackupJob
-        mode: job.mode,
-        reportWhen: (settings && settings.reportWhen) || 'failure',
-      }
-    }
-    if (type === 'metadataBackup') {
-      data = {
-        reportWhen: job.settings['']?.reportWhen ?? 'failure',
-      }
-    }
-
+    const app = this._app
     const logger = this._logger
+    const { id, key, mode, name, settings, type, userId } = job
     const runJobId = logger.notice(`Starting execution of ${id}.`, {
-      data,
+      data:
+        type === 'backup' || type === 'metadataBackup'
+          ? {
+              // $FlowFixMe only defined for BackupJob
+              mode,
+              reportWhen: settings['']?.reportWhen ?? 'failure',
+            }
+          : undefined,
       event: 'job.start',
-      userId: job.userId,
+      userId,
       jobId: id,
-      jobName: job.name,
+      jobName: name,
       scheduleId: schedule?.id,
       // $FlowFixMe only defined for CallJob
-      key: job.key,
+      key,
       type,
     })
 
-    // runId is a temporary property used to check if the report is sent after the server interruption
-    this.updateJob({ id, runId: runJobId })::ignoreErrors()
-    runningJobs[id] = runJobId
-
+    const runningJobs = this._runningJobs
+    const jobIsRunning = id in runningJobs
```

Comment:
Why this var?

Reply:
It's used here https://github.com/vatesfr/xen-orchestra/pull/4534/files#diff-1e08af7fb73e8162884e7e6bd671d48dR330